models/datasource.js

```diff
@@ -0,0 +1,25 @@
+const { DataSource } = require("typeorm");
+
+const teaDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+teaDataSource
+  .initialize()
+  .then(() => {
+    console.log("Data server has been initialized!!");
+  })
+
+  .catch((err) => {
+    console.log(err);
+    console.log("Failed to connect database");
```

Comment:
- 해당 코드는 클라이언트에게 전달하는 것이 아니라 백엔드 개발자가 로그를 확인하기 위해  메시지는 합쳐서 보여주세요!
- 또한 데이터베이스 커넥션이 실패했을 때 커넥션 자체를 해제할 수 있는 destroy() 함수를 추가해 주면 좋을 것 같습니다.
```suggestion
    console.log("Failed to connect database", err);
    teaDataSource.destroy();
```